Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -395,7 +399,7 @@ define([
 
         var cullingVolume = frameState.cullingVolume;
 
-        var boundingVolume = surfaceTile.boundingSphere3D;
+        var boundingVolume = defaultValue(surfaceTile.boundingOBB, surfaceTile.boundingSphere3D);
 
         if (frameState.mode !== SceneMode.SCENE3D) {
```

Comment:
This reminds me - have you tested in 2D and Columbus view?  Make sure to visually inspect too.